godel_simple_gui/src/states/select_all_surface_state.cpp

```diff
@@ -0,0 +1,36 @@
+#include <ros/console.h>
+#include "godel_msgs/SelectSurface.h"
+#include "godel_simple_gui/blending_widget.h"
+#include "godel_simple_gui/states/select_all_surface_state.h"
+#include "godel_simple_gui/states/scan_teach_state.h"
+#include "godel_simple_gui/states/planning_state.h"
+
+void godel_simple_gui::SelectAllSurfaceState::onStart(BlendingWidget& gui)
+{
+  const std::string SELECT_SURFACE_SERVICE = "select_surface";
+  ros::ServiceClient client = gui.nodeHandle().serviceClient<godel_msgs::SelectSurface>(
+      SELECT_SURFACE_SERVICE);
+  godel_msgs::SelectSurface msg;
+  msg.request.action = msg.request.SELECT_ALL;
+  client.call(msg.request, msg.response);
```

Comment:
Should this service call be checked just to make sure that the call was handled properly?